scripts/gen_release_notes/README.md

```diff
@@ -0,0 +1,203 @@
+# this can be omitted if one is ok with this default path
+$ export CONFIG_LOC="path/to/.gen_release.config"
+
+# This token should have repository read access
+$ export GIT_TOKEN="pat_value" 
+```
+
+3. Invoke the command
+
+```bash
+# Help output
+$ genreleasenotes -h
+usage: genreleasenotes [-h] last_release_tag_branch next_release_tag_branch release_changes_branch sprint_number
+
+positional arguments:
+  last_release_tag_branch
+                        tag or branch name of the starting point to check for changes
+  next_release_tag_branch
+                        tag or branch name of the ending point to stop checking for changes
+  release_changes_branch
+                        the name of the branch that is used for determing the changed directories for a release
+  sprint_number         the current sprint number these notes were generated from
+
+optional arguments:
+  -h, --help            show this help message and exit
+
+# Example Usage
+$ genreleasenotes RC/0.28 RC/0.29 ms/DEV-1449-tst 32
+```
+
+## Configuration
+
+The nature of this script is to be as flexible as however many use cases there are. As such, this script has an external configuration file that can alter the behavior of the changelog generation.
+
+Below is a list of all of the configurable values
+
+| Variable     | Description     | Example Value |
+|--------------|-----------------|---------------|
+| tag\_regex | String value to check for tags in a repo. Note that special characters (i.e. `.`) do not need escape characters. |  `"^[0-9]{1,}.[0-9]{1,}.[0-9]{1,}$"` |
+| rc\_regex | String value to check for release candidate (e.g. specific branches) in a repo. Note that special characters (i.e. `.`) do not need escape characters. | `"^RC/[0-9]{1,}.[0-9]{1,}$"` | =
+| bug\_fix\_regex | List of values that correspond to a bug fix. | `["fix","hotfix","bug"]` |
+| main\_branch\_name | Name of the main/master branch in a repo. | `"master"` |
+| tst\_env\_name | Name of test environment. | `"tst"` |
+| prd\_env\_name | Name of production environment. | `"prd"` |
+| dev\_env\_name | Name of development environment. | `"dev"` |
+| misc\_env\_name | Name of general/non-environment specific changes. | `"general"` |
+| root\_env\_dir\_name | Name of the directory that corresponds to the start of the environment roots | `"env"` |
+| repo\_url | Repository to analyze with this script. | `"https://github.com/Nebulaworks/orion"` |
+| url\_repo\_endpoint | URL endpoint that corresponds to what endpoint said `repo_url` corresponds towards. | `"https://github.com"` |
+| pr\_body\_prefix | String to determine what pattern to look for in pull request titles. | `"] -"` |
+| output\_file\_name | Path to output the results. | `"gen_output_results.md"`|
+| weight\_scale | Scale to determine change priority. More details [below](#weight-scale). | `["env": "1", "modules": "10", "docs": "20"]` |
+
+### Weight Scale
```

Comment:
Thanks for explaining the scale I needed a refresher for how this worked myself 😅